CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to the Lethe project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+### [Master] - 2025-08-31
+
+### Changed
+
+- MINOR The rolling resistance models naming convention was redundant. Every option had an extra \"_resistance\" at the end, which was not necessary. Options are now "none", "constant", "viscous" and "epsd". [#1640](https://github.com/chaos-polymtl/lethe/pull/1640)
```

Comment:
```suggestion
- MAJOR The rolling resistance models naming convention was redundant. Every option had an extra \"_resistance\" at the end, which was not necessary. Options are now "none", "constant", "viscous" and "epsd". [#1640](https://github.com/chaos-polymtl/lethe/pull/1640)
```

Reply:
I think we also put a bigger warning when there is a syntax change. Can ctrl+f syntax in the changelog and change? I think any syntax change is a major change